app/Resources/views/full/home.html.twig

```diff
@@ -1,8 +1,21 @@
 {% extends "pagelayout.html.twig" %}
```

Comment:
I did it. But it seems that the parameter is not injected in the template.
`Variable "pagelayout" does not exist in full\home.html.twig`

FYI, ```viewbaseLayout``` is injected